packages/runtime/src/helpers/config.ts

```diff
@@ -1,4 +1,4 @@
-import type { NetlifyConfig } from '@netlify/build'
+import type { NetlifyConfig } from '@netlify/build/types'
```

Comment:
I don't know why, but for `@netlify/build/29.20.15` I had to import from the types folder directly even though the editor/TS language server was able to resolve the types from `@netlify/build`. I was even able to go to definition no problem. IT was only when I compiled that it would complain. Setting it to `@netlify/build/types` made tsc happy.